src/main.ts

```diff
@@ -41,6 +41,8 @@ const main = async () => {
     ? LocalWorkspace.createOrSelectStack(stackArgs, stackOpts)
     : LocalWorkspace.selectStack(stackArgs, stackOpts));
 
+  config.projectName = (await stack.workspace.projectSettings()).name;
```

Comment:
Maybe we should split this into two variables?

```suggestion
	const projectSettings = await stack.workspace.projectSettings();
  config.projectName = projectSettings.name;
```

Something like that? Just to improve readability?

Maybe we should add an invariant as well? I'm not sure if we do any validation in `projectSettings()`, so might not be needed.